### ISSUE TITLE
composer require tattali/calendar-bundle (AdminThemeBundle)

### ISSUE THREAD

**Issue by BenoKop** [tattali/CalendarBundle#10]
Hello,

Impossible for me to fetch the bundle, when I do **composer require tattali/calendar-bundle**, I get the following error:
<pre>
Your GitHub credentials are required to fetch private repository metadata (https://github.com/kevinpapst/AdminThemeBundle)
Head to https://github.com/settings/tokens/new?scopes=repo&description=Composer+on+deckgenius-web+2019-07-01+0703
to retrieve a token. It will be stored in "/home/bkopp/.config/composer/auth.json" for future use by Composer.
</pre>

AdminThemeBundle migrated to AdminLTEBundle ?

Thanks in advance.

**Comment by tattali** [comment]
Hi, I could not reproduce this error. Which version of symfony do you use?

**Comment by tattali** [comment]
Is this error append in the cache clear command? because it's probably not come from this bundle?

**Comment by BenoKop** [comment]
I found the error since but I'm sorry that I didn't update the issue.

I remove this in my composer.json:

<pre>
"repositories": [
    {
        "type": "vcs",
        "url": "https://github.com/kevinpapst/AdminThemeBundle"
    }
]
</pre>

Then I ran a composer install and it passed.
I took over an existing project and it was pretty old, the AdminThemeBundle plugin became private and when I tried to install any package it just crashed because of this ...

Closing it here.
